--- a/ports/mongoose/portfile.cmake
+++ b/ports/mongoose/portfile.cmake
@@ -5,8 +5,8 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO cesanta/mongoose
-    REF 8e520756366ca5739f13dc6ad65fcf269dbbc994 #v7.1
-    SHA512 a966a8b4e47e36da0f630c3cf343c85a1c1138508a82e506b21e4b8bd72573d0e0145318f97b32a67f423e033b348de76a00b780430e4e69d1a98bd7494a3e0a
+    REF d5993ba27ece4b406c230eca63b76dd5d2c28a2d # 7.7
+    SHA512 b5633ae2845d6000549f84dd4933064c8ba6eedc37004666fa2de428e31df2f2ba2422a4434b001629be7bb52b94c52ed62e904231f39fc12074158d85ed145d
     HEAD_REF master
 )
